packages/mds-policy-author/tests/request-handlers.test.ts

```diff
@@ -0,0 +1,41 @@
+import Sinon from 'sinon'
+import db from '@mds-core/mds-db'
+
+import { getPolicies } from '../request-handlers'
+import { mockReq, mockRes } from 'sinon-express-mock'
+
+describe('Policy Author Request Handlers', () => {
+  describe('getPolicies()', () => {
+    afterEach(() => {
+      Sinon.restore()
+    })
+
+    it('Gets all published policies', async () => {
```

Comment:
So these tests just verify that the API calls the appropriate db method?

Reply:
Yes, ideally we'd still have a corresponding integration test but as you know the existing approach is a bit fraught.